backend/src/contexts/users/domain/models/UserDto.ts

```diff
@@ -1,8 +1,11 @@
+import { UserRole } from "./UserRole";
+
 export interface UserDto  {
```

Comment:
maybe a silly question but what does DTO stand for?

Reply:
We use it when we are defining an object that does not have behavior but does define structure, normally for passing data between interfaces. In our case in the repositories.

https://en.wikipedia.org/wiki/Data_transfer_object